Tools/CISupport/ews-build/steps.py

```diff
@@ -768,20 +774,23 @@ def start(self):
             return None
 
         if self._patch_is_relevant(patch, self.getProperty('buildername', '')):
-            self._addToLog('stdio', 'This patch contains relevant changes.')
+            self._addToLog('stdio', 'This {} contains relevant changes.'.format(self.change_type.lower()))
             self.finished(SUCCESS)
             return None
 
-        self._addToLog('stdio', 'This patch does not have relevant changes.')
+        self._addToLog('stdio', 'This {} does not have relevant changes.'.format(self.change_type.lower()))
         self.finished(FAILURE)
         self.build.results = SKIPPED
-        self.build.buildFinished(['Patch {} doesn\'t have relevant changes'.format(self.getProperty('patch_id', ''))], SKIPPED)
+        self.build.buildFinished(['{} {} doesn\'t have relevant changes'.format(
+            self.change_type,
+            self.getProperty('patch_id', '') or self.getProperty('github.number', ''),
+        )], SKIPPED)
         return None
 
 
 class FindModifiedLayoutTests(AnalyzeChange):
     name = 'find-modified-layout-tests'
-    RE_LAYOUT_TEST = br'^(\+\+\+).*(LayoutTests.*\.html)'
+    RE_LAYOUT_TEST = br'^(\+\+\+)?(.*)?(?P<name>LayoutTests.*\.html)'
```

Comment:
This looks unacceptable as this could match a lot more things in a patch workflow. Remember that we are checking the whole content of the patch, if a patch contains a comment saying `// FIXME: update LayoutTests/sometest.html after xyz`, we wouldn't want to consider that as an modified test.

Reply:
Will put this part of the change in a different PR